### ISSUE TITLE
V2G-Liberty does not handle Home Assistant restarts

### ISSUE THREAD

**Issue by rhpijnacker** [V2G-liberty/addon-v2g-liberty#72]
Since I am using HA for other things than just V2G-Liberty, I regularly (have to) restart HA after updating e.g. HACS components.
With every restart V2G-Liberty looses the connection to HA.
It would be nice if it would automatically reconnect without interrupting the dis-/charging session in progress.

**Comment by ArdJonker** [comment]
Thanks for researching this. Did you test already? 
If you get good results i'll add it to the next version.

**Comment by rhpijnacker** [comment]
I did test this, and it looks like adding the plugin directive makes little difference.

**Comment by ArdJonker** [comment]
Should this issue be kept open?

**Comment by rhpijnacker** [comment]
Yes, please. It would nice if V2G-Liberty recovers nicely fromHA restarts.
